### ISSUE TITLE
💡 Feature Request: Display the floor next to the "<playername>'s Dungeon"

### ISSUE THREAD

**Issue by Dx8pi** [DiscordDungeons/Bugs#1314]
### Is your feature request related to a problem?

Like this

Dx8pi's Dungeon - Floor 3

only way to see it currently is by descending and remembering it before moving.

### Describe alternatives you've considered

idk, you can see it when you first enter a new floor, that's the only time tho

**Comment by Willard21** [comment]
`/dungeon status` is a thing

**Comment by Dx8pi** [comment]
Yes, but that clogs up the chat, so you eventually have to scroll up to press the buttons. Ideally I would want the dungeon board at the bottom of all the messages as often as possible. I believe other players would agree with me. Besides, it's not that difficult of an addition to make, no?
